### ISSUE TITLE
BadZipFile - zip file no longer exist 

### ISSUE THREAD

**Issue by bairo** [re-search/DocProduct#3]
it seems that the zip file no longer exist in the onedrive link shared in notebook. can you please have a look

![image](https://user-images.githubusercontent.com/3649124/60153740-c6e3d300-9817-11e9-9dab-d92b98cfd9d2.png)

**Comment by lmcdo** [comment]
Is there any progress on this please? Very grateful, if so

**Comment by Santosh-Gupta** [comment]
Looking into it. Can you link the notebook that is giving you the issue?

**Comment by pumpkinband** [comment]
same error, did anybody solved it ??

**Comment by jayeshdesai4520** [comment]
I got the same error here is the link of the notebook [https://colab.research.google.com/drive/11hAr1qo7VCSmIjWREFwyTFblU2LVeh1R]

**Comment by Santosh-Gupta** [comment]
Still working on it, somehow our checkpoint file got deleted from our Onedrive account, working to get a new one up ASAP

**Comment by Santosh-Gupta** [comment]
It looks like the file may have gone down due to many new downloads due to the boost from Siraj's video. It's back up now. 

Try it now, let me know if it works.

**Comment by pumpkinband** [comment]
yup it's working thanks @Santosh-Gupta

**Comment by JayYip** [comment]
Closing the issue since it's fixed.

**Comment by jayeshdesai4520** [comment]
again got the same error here is the link of notebook https://colab.research.google.com/drive/11hAr1qo7VCSmIjWREFwyTFblU2LVeh1R

**Comment by JayYip** [comment]
> again got the same error here is the link of notebook https://colab.research.google.com/drive/11hAr1qo7VCSmIjWREFwyTFblU2LVeh1R

Yes. This file is hosted in my personal OneDrive account. Seems downloading from colab is not reliable. I'll re-open the issue. We need a better place to store the file. 

As a work around, you can download [this file](https://onedrive.live.com/download?cid=9DEDF3C1E2D7E77F&resid=9DEDF3C1E2D7E77F%2132792&authkey=AEQ8GtkcDbe3K98) in the browser and then upload it to colab. 

Sorry for the inconvenience that may cause.

**Comment by tahercoolguy** [comment]
@JayYip 
After opening link which you have provided . I am getting "This item might not exist or is no longer available ".  kindly upload again zip file to get worked thank you

**Comment by lmcdo** [comment]
Yes, it works now, thank you Santosh, and needless to say, brilliant work!!!

**Comment by JayYip** [comment]
@tahercoolguy Are you still having the same issue?

**Comment by davidbetancur8** [comment]
Its down again!

**Comment by Santosh-Gupta** [comment]
It looks like our One Drive can't handle the popularity post-siraj vid. It'll go back up again but eventually will come down. If you're able to download the file, I very much recommend save it to your own google drive so you'll always have it, until we can find a more robust way to store the files.

**Comment by jsem2k1** [comment]
It's down again...

**Comment by Santosh-Gupta** [comment]
Try again in a few hours, and if you're able to download the files, save it to your own Google Drive so that you'll always have access to it, because it'll go down again due to number of people trying out the colab demos.

**Comment by JayYip** [comment]
We're having a hard time keeping the file available to everybody. We don't have budget to host the checkpoint on cloud server. If anybody can provide some help on this, please make a PR! Thanks!

**Comment by shravankumar9892** [comment]
Hello all, Shravan here. Can someone please [email me](mailto:shravankumarshetty9892@gmail.com) the link to the checkpoint file, I wanted to experiment with DocProduct. Or else @JayYip or @Santosh-Gupta can you tell me any alternative way to start exploring the project with considerable correctness in the output?

**Comment by Santosh-Gupta** [comment]
It would still be the Onedrive file. For now we recommend trying every couple hours, and once you are able to get the weights, save it to your computer of cloud.
